### ISSUE TITLE
use pretrain model inference the test wav, i got overload wav.

### ISSUE THREAD

**Issue by haha010508** [xi-j/Mamba-TasNet#5]
use pretrain model inference the test wav, i got overload wav, like this, please help fix it, thanks! i use this model:dpmamba_M.yaml
[frank_t.zip](https://github.com/user-attachments/files/17629099/frank_t.zip)

**Comment by xi-j** [comment]
Hi, frank_test.wav sounds very distorted. Can you double check this is your input?

**Comment by haha010508** [comment]
yes, i check the input is right, my code and the result are in file, please help doule check, ths!
[frank_file.zip](https://github.com/user-attachments/files/17641225/frank_file.zip)
and this is my pip install list
Package            Version
------------------ -----------
attrs              24.2.0
Automat            24.8.1
buildtools         1.0.6
causal-conv1d      1.1.3.post1
certifi            2024.8.30
charset-normalizer 3.4.0
click              8.1.7
constantly         23.10.4
docker-pycreds     0.4.0
docopt             0.6.2
einops             0.8.0
filelock           3.13.1
fsspec             2024.2.0
furl               2.1.3
future             1.0.0
gitdb              4.0.11
GitPython          3.1.43
greenlet           3.1.1
huggingface-hub    0.26.2
hyperlink          21.0.0
HyperPyYAML        1.2.2
idna               3.10
incremental        24.7.2
Jinja2             3.1.3
joblib             1.4.2
mamba-ssm          1.1.3.post1
MarkupSafe         2.1.5
mir_eval           0.6
mpmath             1.3.0
networkx           3.2.1
ninja              1.11.1.1
numpy              1.26.3
orderedmultidict   1.0.1
packaging          24.1
pillow             11.0.0
pip                24.3.1
platformdirs       4.3.6
protobuf           5.28.3
psutil             6.1.0
python-dateutil    2.9.0.post0
PyYAML             6.0.2
redo               3.0.0
regex              2024.9.11
requests           2.32.3
ruamel.yaml        0.18.6
ruamel.yaml.clib   0.2.12
safetensors        0.4.5
scipy              1.13.1
sentencepiece      0.2.0
sentry-sdk         2.17.0
setproctitle       1.3.3
setuptools         75.3.0
simplejson         3.19.3
six                1.16.0
smmap              5.0.1
speechbrain        1.0.0
SQLAlchemy         2.0.36
sympy              1.13.1
tokenizers         0.20.1
tomli              2.0.2
torch              2.1.0+cu121
torchaudio         2.1.0+cu121
torchvision        0.16.0
tqdm               4.66.6
transformers       4.46.1
triton             2.1.0
Twisted            24.10.0
typing_extensions  4.9.0
urllib3            2.2.3
wandb              0.18.5
wheel              0.44.0
zope.interface     7.1.1

**Comment by haha010508** [comment]
update: need norm for out, closed
